ask-sdk-model-runtime/ask_sdk_model_runtime/lwa/access_token_request.py

```diff
@@ -34,25 +34,23 @@ class AccessTokenRequest(object):
     :param scope: The required scope for which the access token is
         requested for
     :type scope: str
-    :param refresh_token: Client refresh_token required to get access token for API calls.
+    :param refresh_token: Client refresh_token required to get access token
+        for API calls.
     :type refresh_token: str
     """
-    deserialized_types = {
-        'client_id': 'str',
-        'client_secret': 'str',
-        'refresh_token': 'str',
-        'scope': 'str'
-    }
-
-    attribute_map = {
-        'client_id': 'client_id',
-        'client_secret': 'client_secret',
-        'refresh_token': 'refresh_token',
-        'scope': 'scope'
-    }
-
-    def __init__(self, client_id=None, client_secret=None, scope=None, refresh_token=None):
-        # type: (Optional[str], Optional[str], Optional[str], Optional[str]) -> None
+    deserialized_types = {'client_id': 'str', 'client_secret': 'str',
+        'refresh_token': 'str', 'scope': 'str'}
+
+    attribute_map = {'client_id': 'client_id',
+        'client_secret': 'client_secret', 'refresh_token': 'refresh_token',
+        'scope': 'scope'}
+
+    def __init__(self, client_id=None,  # type: Optional[str]
+                 client_secret=None,  # type: Optional[str]
+                 scope=None,  # type: Optional[str]
+                 refresh_token=None  # type: Optional[str]
+                 ):
+        # type: (...) -> None
```

Comment:
Same for the type hints here, as described above.

Reply:
Ok, will revert it back to inline type comments